File: Python/Human.py
# Saya Ananda Myzza Marhelio NIM 2100702 mengerjakan soal Latihan 4 dalam mata kuliah Desain Pemrograman Berorientasi
# Objek untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

class Human:
    # Attribut private
    __NIK = ""
    __name = ""
    __gender = ""

    # Constructor
    def __init__(self, NIK="", name="", gender=""):
        self.__NIK = NIK
        self.__name = name
        self.__gender = gender

    # Setter dan Getter
    # Setter & Getter untuk NIK.
    def getNIK(self):
        return self.__NIK

    def setNIK(self, NIK):
        self.__NIK = NIK

    # Setter & Getter untuk Nama.
    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    # Setter & Getter untuk Gender.
    def getGender(self):
        return self.__gender

    def setGender(self, gender):
        self.__gender = gender

File: C++/Human.cpp
/* Saya Ananda Myzza Marhelio NIM 2100702 mengerjakan soal Latihan 4 dalam mata kuliah Desain Pemrograman Berorientasi
Objek untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. */

#include "Header.hh"

/* Constructor */
Human::Human()
{
    this->NIK = "";
    this->name = "";
    this->gender = "";
}
Human::Human(string NIK, string name, string gender)
{
    this->NIK = NIK;
    this->name = name;
    this->gender = gender;
}

/* Setter & Getter untuk NIK */
string Human::getNIK()
{
    return this->NIK;
}
void Human::setNIK(string NIK)
{
    this->NIK = NIK;
}

/* Setter & Getter untuk Nama */
string Human::getName()
{
    return this->name;
}
void Human::setName(string name)
{
    this->name = name;
}

/* Setter & Getter untuk Gender */
string Human::getGender()
{
    return this->gender;
}
void Human::setGender(string gender)
{
    this->gender = gender;
}

/* Destructor */
Human::~Human()
{
}
